### ISSUE TITLE
Server Files Contain "Modpack Configuration Checker"

### ISSUE THREAD

**Issue by Valericoe** [Sunekaer/stoneBlock#291]
## Bug Report
Server crashes upon starting. This is due to "Modpack Configuration Checker" not being compatible with (Linux?) servers.
It's a pretty minor thing, but some people may get caught on it.

## Steps to Reproduce (for bugs)
1. Download the server files. (Checked 1.0.17 & 1.0.18)
2. Attempt to start a Linux server

## Logs
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Modpack Config Checker (concheckrmd)
Caused by: java.awt.HeadlessException: 
No X11 DISPLAY variable was set, but this program performed an operation which requires it.

## Fix
Delete "Modpack Configuration Checker" from your server files.

**Comment by Sunekaer** [comment]
My dev server is Linux and it runs fine there

**Comment by Sunekaer** [comment]
You are using Ubuntu so i invite you to read about X and X.org : https://en.wikipedia.org/wiki/X_Window_System .
TL;DR : X.org is an application running on your computer allowing graphical applications to be displayed.

In general, if you are running applications on a machine which has no display (headless machines and/or servers for example), some programs will still try to display something by default, and fail while trying to reach the X11 compatible server application (X.org) running on said machine.
Leading to this line: Can't connect to X11 window server using 'localhost:11.0' as the value of the DISPLAY variable.

The DISPLAY environment variable allows to manually tell programs which display server they should use, and you can use it to point the application to a correct display server should one be present on the machine you are trying to use.
The second option is to tell the program not to display anything, and in the case of a minecraft server translate to the nogui command line option.

**Comment by Valericoe** [comment]
Quick clarification: I'm renting the server, and couldn't start a StoneBlock server without removing the .jar file.

I merely posted this as a suggestion to remove it from the server pack as it doesn't seem to do anything server side (?), but I'm not a modder and wouldn't know.
Also to help anyone with similar issues.
